clang/lib/Sema/SemaExpr.cpp

```diff
@@ -13958,7 +13979,11 @@ QualType Sema::CheckAssignmentOperands(Expr *LHSExpr, ExprResult &RHS,
           LHSType->isObjCObjectPointerType())))
       ConvTy = AssignConvertType::Compatible;
 
+<<<<<<< HEAD
```

Comment:
bad merge!  Below too

Reply:
Yeah, I just hit that error in my compilation. Oops :-D